### PR TITLE
Reducers and actions

### PR DESCRIPTION
**Pull request by ndayishimiyeeric**
### This PR has the following features:

- Structure application files using a "feature folder" approach.
- Book's actions and reducer.
- Categories actions and reducer.
- Configure Redux Store.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// ACTIONS
+const ADD = 'add_book';
+const REMOVE = 'remove-book';
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x] In keeping with the ducks pattern `(app/reducer/ACTION_TYPE)`, kindly rename your action types here: For example `const ADD = bookstore/books/ADD`.

- [x] **[OPTIONAL]** Wouldn't it be better to rename the action types to something more descriptive and apparent like `ADD_BOOK` and `REMOVE_BOOK`? In the end you can have something like so: `const ADD_BOOK = bookstore/books/ADD_BOOK`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// ACTIONS
+const MSG = 'categories_under_construction';
```
Comment (TedLivist) [inline src/redux/categories/categories.js]: - [x] Kindly rename your action type in keeping with the ducks pattern.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ndayishimiyeeric,

Good job so far!
✔️ Set up of store
✔️ Set up of reducers
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+// ACTIONS
+const ADD_BOOK = 'bookstore/books/ADD';
+const REMOVE_BOOK = 'bookstore/books/REMOVE';
```
Comment (Mre55) [inline src/redux/books/books.js:3]: **[OPTIONAL]** Good job 👍 implementing the changes requested but the previous reviewer but there are a few things that can also be updated as suggested by the previous reviewer I highly recommend changing these two lines to something like this:

```
const ADD_BOOK = bookstore/books/ADD_BOOK
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

**Review by Mre55 — APPROVED** [review]
**Hi @ndayishimiyeeric,**

# Status: APPROVED 🏆 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**